src/src/PluginStructs/P077_data_struct.cpp

```diff
@@ -163,4 +163,56 @@ bool P077_data_struct::processSerialData() {
   return found;
 }
 
+/**
+ * plugin_write: Handle commands
+ * csereset: reset calibration values
+ * csecalibrate,[voltage],[current],[power]: set calibration values, 0 value(s) will be skipped
+ * Saves (all) settings if a calibration value is updated, or csereset is used
+ */
+bool P077_data_struct::plugin_write(struct EventStruct *event,
+                                    String              string) {
+  const String cmd = parseString(string, 1);
+  bool success     = false;
+  bool changed     = false;
+
+  if (cmd.equals(F("csereset"))) { // Reset to defaults
+    PCONFIG(0) = HLW_UREF_PULSE;
```

Comment:
HLW_xxx ?
This is the CSE7766 plugin :)

Reply:
That's how it already was in the `P077_data_struct.h` file...:
https://github.com/letscontrolit/ESPEasy/pull/4501/files#diff-919d870c2dd6ade123b7bb2934a14fbbff1067d1a0018b15f6fa035e50d61542L12-L15

I can rename that, of course 😃